WordPress/Classes/ViewRelated/Jetpack Branding/JetpackBannerView.swift

```diff
@@ -0,0 +1,73 @@
+import UIKit
+
+class JetpackBannerView: UIView {
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        setup()
+
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+        setup()
+    }
+
+    func setup() {
+        guard AppConfiguration.isWordPress else {
```

Comment:
Not sure if the responsibility of where the view is used/configured should be of the view itself, what do you think?

Reply:
That's a good point, I'll see if it's feasible to move the logic here to the Stats controller. Doing so might create duplication if this is used in other areas, but I'm also wary of having the logic in the view itself.

Reply:
I moved the logic to the Stats dashboard view controller 👍